Code/Framework/AzCore/AzCore/Math/Geometry3DUtils.cpp

```diff
@@ -27,15 +27,16 @@ namespace AZ::Geometry3dUtils
                 return;
             }
 
-            // For each triangle that's passed in, split it into 4 triangles and recursively subdivide those.
-            //           a
-            //           /\
-            //          /  \
-            //     ab  /----\  ca
-            //        / \  / \
-            //       /   \/   \
-            //    b ------------ c
-            //           bc
+            /** For each triangle that's passed in, split it into 4 triangles and recursively subdivide those.
+             *           a
+             *           /\
+             *          /  \
+             *     ab  /----\  ca
+             *        / \  / \
+             *       /   \/   \
+             *    b ------------ c
+             *           bc
+             */
```

Comment:
Interesting that you found this, it only gets reported with gcc at the moment

Reply:
I saw it in the nightly build :)